flag.go

```diff
@@ -73,26 +73,6 @@ var FlagEnvHinter FlagEnvHintFunc = withEnvHint
 // details. This is used by the default FlagStringer.
 var FlagFileHinter FlagFileHintFunc = withFileHint
 
-// FlagsByName is a slice of Flag.
-type FlagsByName []Flag
```

Comment:
CI error is from here https://github.com/urfave/cli/actions/runs/11450130089/job/31857025546?pr=1988#step:7:111

https://github.com/urfave/cli/blob/5053ec708a44d766e62f7e33baae742b6e2c003c/docs/v3/examples/flags.md?plain=1#L291-L363

This example is using `cli.FlagsByName`.